### DISCUSSION TITLE
How long will it take to respond to each ticket?

### DISCUSSION THREAD

**Discussion by yiyuxin** [community/community#34769] [Copilot Conversations] [answered]
I am using github copilot for the first time , after the trial peiod has ended i forgot to cancel the subscription and my card got charged for yearly subscription , so my question is how can i cancel this yearly subscription and is there a refund when canceling ?

**Comment by iamcjwilliams** [comment, marked as answer]
@yiyuxin Visit here to [cancel your subscription](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-subscription#canceling-your-github-copilot-subscription). To receive a refund you, please open a ticket on our [Support page](https://support.github.com/contact). Our copilot team is currently experiencing a high volume of tickets, so responses will be delayed. Thank you for your patience
